MinimedKit/PumpEvents/PumpAlarmPumpEvent.swift

```diff
@@ -8,11 +8,22 @@
 
 import Foundation
 
+public enum PumpAlarmType: Int {
+    case BatteryOutLimitExceeded = 3
+    case NoDelivery              = 4
+    case BatteryDepleted         = 5
+    case DeviceReset             = 16
+    case ReprogramError          = 61
+    case EmptyReservoir          = 62
+    case UnknownType             = -1
```

Comment:
I tried this:
```
public enum PumpAlarmType: Int {
    case BatteryOutLimitExceeded = 3
    case NoDelivery              = 4
    case BatteryDepleted         = 5
    case DeviceReset             = 16
    case ReprogramError          = 61
    case EmptyReservoir          = 62
    case UnknownType(Int)        = -1
}
```
But I get this error: `Enum with raw type cannot have cases with arguments`.

Not sure that making all of them have an argument is very clean either.